exchange_info_service_test.go

```diff
@@ -49,7 +49,7 @@ func (s *exchangeInfoServiceTestSuite) TestExchangeInfo() {
 				QuotePrecision:     8,
 				OrderTypes:         []string{"LIMIT", "LIMIT_MAKER", "MARKET", "STOP_LOSS_LIMIT", "TAKE_PROFIT_LIMIT"},
 				IcebergAllowed:     true,
-				Filters: []map[string]string{
+				Filters: []map[string]interface{}{
 					{"filterType": "PRICE_FILTER", "minPrice": "0.00000100", "maxPrice": "100000.00000000", "tickSize": "0.00000100"},
 					{"filterType": "LOT_SIZE", "minQty": "0.00100000", "maxQty": "100000.00000000", "stepSize": "0.00100000"},
 					{"filterType": "MIN_NOTIONAL", "minNotional": "0.00100000"},
```

Comment:
thanks @cjamessutton . #58 is working on this as well.
btw, could you pls add more filters with integer value to the test case as in https://github.com/binance-exchange/binance-official-api-docs/blob/master/rest-api.md#symbol-filters, so that we can cover this change in unit test, thanks.

Reply:
Done

Reply:
thanks @cjamessutton